Base on meta/recipes-extended/cups/cups/0001-don-t-try-to-run-generated-binaries.patch

Modify patch file for cups from debian's source code.

diff --git a/ppdc/Makefile b/ppdc/Makefile
index 56e754e..e4bd332 100644
--- a/ppdc/Makefile
+++ b/ppdc/Makefile
@@ -245,8 +245,8 @@ genstrings:		genstrings.o libcupsppdc.so ../cups/$(LIBCUPSSTATIC) \
 	$(CXX) $(ARCHFLAGS) $(LDFLAGS) -o genstrings genstrings.o \
 		-lcupsppdc ../cups/$(LIBCUPSSTATIC) $(LIBGSSAPI) $(SSLLIBS) \
 		$(DNSSDLIBS) $(COMMONLIBS) $(LIBZ)
-	echo Generating localization strings...
-	LD_LIBRARY_PATH=.:../cups/ ./genstrings >sample.c
+#	echo Generating localization strings...
+#	LD_LIBRARY_PATH=.:../cups/ ./genstrings >sample.c
 
 
 #
@@ -263,9 +263,9 @@ ppdc-static:		ppdc.o libcupsppdc.a ../cups/$(LIBCUPSSTATIC) foo.drv foo-fr.po
 	$(CXX) $(ARCHFLAGS) $(LDFLAGS) -o ppdc-static ppdc.o -lcupsppdc \
 		../cups/$(LIBCUPSSTATIC) $(LIBGSSAPI) $(SSLLIBS) $(DNSSDLIBS) \
 		$(COMMONLIBS) $(LIBZ)
-	echo Testing PPD compiler...
-	LD_LIBRARY_PATH=.:../cups/ ./ppdc-static -l en,fr -I ../data foo.drv
-	LD_LIBRARY_PATH=.:../cups/ ./ppdc-static -l en,fr -z -I ../data foo.drv
+#	echo Testing PPD compiler...
+#	LD_LIBRARY_PATH=.:../cups/ ./ppdc-static -l en,fr -I ../data foo.drv
+#	LD_LIBRARY_PATH=.:../cups/ ./ppdc-static -l en,fr -z -I ../data foo.drv
 
 
 #
@@ -291,17 +291,17 @@ ppdi-static:		ppdc-static ppdi.o libcupsppdc.a  ../cups/$(LIBCUPSSTATIC)
 	$(CXX) $(ARCHFLAGS) $(LDFLAGS) -o ppdi-static ppdi.o -lcupsppdc \
 		../cups/$(LIBCUPSSTATIC) $(LIBGSSAPI) $(SSLLIBS) $(DNSSDLIBS) \
 		$(COMMONLIBS) $(LIBZ)
-	echo Testing PPD importer...
-	$(RM) -r ppd ppd2 sample-import.drv
-	LD_LIBRARY_PATH=.:../cups/ ./ppdc-static -I ../data sample.drv
-	LD_LIBRARY_PATH=.:../cups/ ./ppdi-static -I ../data -o sample-import.drv ppd/*
-	LD_LIBRARY_PATH=.:../cups/ ./ppdc-static -I ../data -d ppd2 sample-import.drv
-	if diff -r ppd ppd2 >/dev/null; then \
-		echo PPD import OK; \
-	else \
-		echo PPD import FAILED; \
-		exit 1; \
-	fi
+#	echo Testing PPD importer...
+#	$(RM) -r ppd ppd2 sample-import.drv
+#	LD_LIBRARY_PATH=.:../cups/ ./ppdc-static -I ../data sample.drv
+#	LD_LIBRARY_PATH=.:../cups/ ./ppdi-static -I ../data -o sample-import.drv ppd/*
+#	LD_LIBRARY_PATH=.:../cups/ ./ppdc-static -I ../data -d ppd2 sample-import.drv
+#	if diff -r ppd ppd2 >/dev/null; then \
+#		echo PPD import OK; \
+#	else \
+#		echo PPD import FAILED; \
+#		exit 1; \
+#	fi
 
 
 #
